public/js/icinga/behavior/modal.js

```diff
@@ -187,6 +192,10 @@
      * @param $modal {jQuery} The modal element
      */
     Modal.prototype.hide = function($modal) {
+        var actionUrl = $modal.find('[action]').attr('action');
+        actionUrl = actionUrl.split('&').slice(0, -1).join(" ");
+        $('a[href="' + actionUrl + '"]').removeAttr("style");
```

Comment:
Eeh.. I think now it's the time to consider storing a reference to the modal opener temporarily on the instance while the modal is open. There's just a single modal at a time after all.

Also, don't use jQuery to reset the style. You already didn't need it to define it.